sale_stock_restocking_fee_invoicing/models/stock_move.py

```diff
@@ -36,3 +34,12 @@ def action_done(self):
                 self.browse(move_ids_to_charge)
             )
         return result
+
+    @api.multi
+    def _append_move_to_charge(self):
```

Comment:
@lmarion-source Thank you for this extension hook. Can you rename the method '_is_restocking_fee_chargeable`